packages/devextreme/js/ui/validator.js

```diff
@@ -134,10 +134,18 @@ const Validator = DOMComponent.inherit({
             const isRequired = rules.some(({ type }) => type === 'required') || null;
 
             if(dxStandardEditor.isInitialized()) {
-                dxStandardEditor.setAria('required', isRequired);
+                dxStandardEditor.setAria({
+                    'required': isRequired,
+                    'invalid': isRequired,
+                });
             }
-            dxStandardEditor.option('_onMarkupRendered', () => {
-                dxStandardEditor.setAria('required', isRequired);
+            dxStandardEditor.option('_onMarkupRendered', (data) => {
+                const isInvalid = data === '';
+
+                dxStandardEditor.setAria({
+                    'required': isRequired,
+                    'invalid': isInvalid,
```

Comment:
With your logic, it is possible to have invalid: true (if data is empty) even if required is false. Please check with the AXE validator if this combination will error out. I think we should have a check here that sets invalid to true in case of an empty value only if required is true. Please add a test for this

Reply:
please check this functionality in vue also